dom0/sd-logging-setup.sls

```diff
@@ -14,15 +14,12 @@ install-securedrop-log-package:
       - sls: fpf-apt-test-repo
 {% endif %}
 
-{% if grains['id'] == "sd-log-buster-template" %}
+{% if grains['id'] in ["sd-small-buster-template", "sd-large-buster-template"] %}
```

Comment:
Do we need to install redis in large template? Should only be required for _receiver_ of logs.